.github/workflows/ci.yml

```diff
@@ -93,6 +96,10 @@ jobs:
         !contains(matrix.platform.target, 'wasm32'))
       run: cargo $CMD test --verbose --target ${{ matrix.platform.target }} $OPTIONS --features $FEATURES
 
+    - name: Lint with clippy
+      shell: bash
+      if: matrix.rust_version != 'nightly'
+      run: echo $FEATURES && cargo clippy --all-targets --target ${{ matrix.platform.target }} $OPTIONS --features $FEATURES
```

Comment:
Do we need to `echo` what features are available? That should already be part of the `job` title because this is part of the `strategy: matrix:`?

Reply:
I just added it to test, since I want to understand how to blacklist the run on non-default-features.